recipes-wpe/wpeframework/include/spark.inc

```diff
@@ -0,0 +1,28 @@
+# Provides configuration for the Cobalt Plugin
+#WPE_SPARK_PLATFORM ??= "${@bb.utils.contains('DISTRO_FEATURES', 'wayland', ' -DPLATFORM_WAYLAND_EGL=ON', '', d)}"
```

Comment:
Is this needed? If not please remove. If yes then it needs attention to wrap it up

Reply:
removed